packages/identity/src/verifySignature.ts

```diff
@@ -12,3 +12,14 @@ export const verifySignature = async (
   const algorithm = getAlgorithmParameters(config.signAlg, 'verify')
   return await crypto.verify(algorithm.algorithm as Algorithm, publicKey, signature, Buffer.from(message))
 }
+
+export const verifyDataSignature = async (
+  signature: ArrayBuffer,
+  data: Uint8Array,
```

Comment:
As the only difference here is data type, why not to extend the type and keep it a single method? 
```
data: string | Uint8Array
```

Reply:
I personally think it's best to do it now. Otherwise the extra issue will get low priority and it's likely we'll postpone it until we forget it